formats/contact/publisher/details.json

```diff
@@ -0,0 +1,330 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "title",
+    "description"
+  ],
+  "properties": {
+    "slug": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "organisation": {
```

Comment:
Good point for the future. "The future" here is because I've just regained a decent thought process for this after talking it through and having been away from it for a few days. The Contacts app is already on V1, but it is schemaless, so we should be documenting how it currently does things on V1 (as evidenced by this schema with an expanded "organisation" part), not how we want it to look in the next story which we've not got to yet - actually doing the work to make it V2. How can we invent schemas for things we haven't made yet and make everyone happy? Other apps don't have this problem as much 'cause they're going from **nothing `publishing-api`-related** to **V2**, AFAICT.